source/framework/core/inc/TRestEveEventViewer.h

```diff
@@ -36,10 +27,22 @@
 #include <TEveText.h>
 #include <TEveTrans.h>
 #include <TEveViewer.h>
+#include <TGLViewer.h>
 #include <TGTab.h>
+#include <TGeoManager.h>
+#include <TGeoMaterial.h>
 #include <TGeoNode.h>
+#include <TObject.h>
+#include <TPad.h>
+#include <TVector3.h>
 
+#include "TRestEventViewer.h"
+
+#if ROOT_VERSION_CODE >= ROOT_VERSION(6, 24, 0)
+#define GEOM_SCALE 1
+#else
 #define GEOM_SCALE 0.1
```

Comment:
I don't remember about this.

Looks like GEOM_SCALE is an scale factor to display the deposited energy. and it is just used here:
TRestEveEventViewer::AddSphericalHit(double x, double y, double z, double r
adius, double en) {

I would suggest to remove the define and either remove GEOM_SCALE or add an rml variable about that. Perhaps, not in this pull request.

Reply:
Why would you add an RML variable? this is just a constant, it shouldn't be modified by the user. It appears because you need to rescale all positions by 0.1 in order for them to be in line with the displayed volume by `TGeoManager`. This has always seemed off to me, but its nothing related to REST (In any ROOT macro you also need to divide by 10).

I guess they caught up to this "bug" and fixed it in the newest versions, so you no longer need to do this, thats why I implemented this logic.

Reply:
Well, to add an RML variable is not required. But there is no reason to have a #define inside a header which is only used in TRestEveEventViewer::AddSphericalHit(double x, double y, double z, double r
adius, double en)

Can you just move this hunk:
#if ROOT_VERSION_CODE >= ROOT_VERSION(6, 24, 0)
#define GEOM_SCALE 1
#else
#define GEOM_SCALE 0.1
inside TRestEveEventViewer::AddSphericalHit